### ISSUE TITLE
Can this method be used for Conv3d layer?

### ISSUE THREAD

**Issue by Fangwq** [henrygouk/keras-lipschitz-networks#1]
Can this method be used for Conv3d layer and enforce Lipschitz Continuity?
And how to do it? Thank you very much!

**Comment by henrygouk** [comment]
Yes it can. It might be a bit hard to think about how to do it for L1 and L_\infty norms, but for the L2 norm you can just adapt the lines 102-110 of `arch/lipschitz.py` and replace the `K.conv2d` and `K.conv2d_transpose` operations with their conv3d versions.

**Comment by Fangwq** [comment]
Thank you very much!
